.changeset/slow-rats-jog.md

```diff
@@ -0,0 +1,6 @@
+---
+"@eth-optimism/integration-tests": patch
+"@eth-optimism/data-transport-layer": patch
+---
+
+linted and fixed
```

Comment:
Do we really need to patch with linting?

Reply:
probably not! i thought this was needed haha but can remove

Reply:
Yeah no need to changeset if there's no code that needs to be released :)